packages/x-charts/src/ChartsLegend/ChartsLegend.tsx

```diff
@@ -138,6 +139,14 @@ ChartsLegend.propTypes = {
    * @default 5
    */
   markGap: PropTypes.number,
+  /**
+   * Callback fired when a legend item is clicked.
+   * @param {React.MouseEvent<SVGRectElement, MouseEvent>} event The click event.
+   * @param {SeriesLegendItemContext} legendItem The legend item data.
+   * @param {number} index The index of the clicked legend item.
+   * @default undefined
```

Comment:
```suggestion
```

Usually don't put `undefined` as a default value since it's the default "default value"